pdp/static/js/pdp_controls.js

```diff
@@ -290,6 +297,36 @@ RasterDownloadLink.prototype = {
         );
         return url;
     },
+    getSingleCellDownloadLink: function (bounds, feature) {
+        if (this.ext !== "aig") {
```

Comment:
Since the aig format is the exceptional case, it's sometimes a little cleaner to make a *positive* test for the aig format, and then return. Then the bulk of the method's code isn't stuck in an `if` branch.